05-AdvancedTopics/web-search-mcp/README.md

```diff
@@ -0,0 +1,214 @@
+# Web Search MCP Server
```

Comment:
Please follow the headers we have for lessons, see h1, h2 and h3 headers, https://github.com/microsoft/mcp-for-beginners/blob/main/03-GettingStarted/01-first-server/README.md e.g Overview, Learning Objectives etc

But yea, appreciate the effort :)